src/handformer.cpp

```diff
@@ -1,33 +1,42 @@
 #include "handformer.h"
 
-#include <ctype.h>
+#include <ctype.h>  // for isdigit, isalpha
 
-#include <algorithm>
-#include <cstdint>
-#include <iterator>
-#include <map>
-#include <string>
-#include <vector>
+#include <algorithm>  // for transform
+#include <cstdint>    // for uint8_t, int8_t
+#include <iterator>   // for back_insert_iterator, back_inserter
+#include <map>        // for map, operator==, _Rb_tree_iterator, _Rb_tree_...
+#include <string>     // for basic_string, string, to_string
+#include <utility>    // for pair
+#include <vector>     // for vector
```

Comment:
I personally prefer no comments on the headers as it has a tendency to get out of sync, but not a big deal.

Reply:
fixed in 5ee5d6df3ae88121f47d7fc03d03b9139bcb821f